tools/check-commit-messages/check_commit_messages.py

```diff
@@ -0,0 +1,73 @@
+"""
+Copyright (c) 2024, Zededa Inc.
+SPDX-License-Identifier: Apache-2.0
+
+This script checks the commit messages in the current branch to ensure they follow the
+recommended format. The recommended format is as follows:
+* Commit message should have a subject and body
+* An empty line should separate the subject and body
+* The body should not be empty (it should contain more than just "Signed-off-by")
```

Comment:
Should we also impose line length limit?

Reply:
I thought about it, but we have too many people for whom even these rules can be too strict =D So, let's introduce the restrictions step by step.